src/Eloquent/Model.php

```diff
@@ -71,11 +71,11 @@ public function newEloquentBuilder($query)
     }
 
     /**
-     * //TODO: Dusan HELPPPPPPPPP
+     * Create a new Scylla Timestamp with the current time
      */
     public function freshTimestamp()
     {
-        return new Timestamp();
+        return new Timestamp(time(), 0);
```

Comment:
It would be good to use name parameters here. I have no idea what both parameters really are.

Reply:
Thx! Done.